build.gradle

```diff
@@ -25,7 +20,7 @@ ext {
     androidxLegacySupportV13Version = '1.0.0'
     androidxMediaVersion = '1.2.1'
     androidxVectorDrawableAnimatedVersion = '1.1.0'
-    daggerVersion = '2.42'
+    daggerVersion = '2.46.1'
```

Comment:
If I don't bump version to the same version used in wpandroid, the build fails with cryptic errors